plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/SwtBotAppEngineActions.java

```diff
@@ -139,8 +139,16 @@ public static IProject importNativeProject(SWTWorkbenchBot bot, String projectNa
 
     SwtBotTreeUtilities.waitUntilTreeHasItems(bot, bot.tree());
     SWTBotTreeItem treeItem = bot.tree().expandNode("General");
-    SwtBotTreeUtilities.waitUntilTreeItemhasChild(bot, treeItem,
-        "Existing Projects into Workspace");
+    try {
+      SwtBotTreeUtilities.waitUntilTreeItemhasChild(bot, treeItem,
+          "Existing Projects into Workspace");
+    } catch (TimeoutException e) {
+      System.out.println(treeItem + ": expanded? " + treeItem.isExpanded());
```

Comment:
These seem like useful diagnostics that should be moved into `waitUntilTreeItemHasChild`.  Oh, and the method should have a capitalized `H`.